--- gnash-0.8.4/libbase/Makefile.am.orig	2008-10-12 15:03:29.000000000 +0200
+++ gnash-0.8.4/libbase/Makefile.am	2008-10-15 21:21:50.866638174 +0200
@@ -20,14 +20,6 @@
 # this is where Gnash plugins get installed
 pluginsdir = $(prefix)/lib/gnash/plugins
 
-if INSTALL_LTDL
-if !LIBLTDL2
-include_HEADERS =  $(top_srcdir)/libltdl/ltdl.h
-LIBLTDLLIB = libltdl.la
-LIBLTDLHEAD =  $(top_srcdir)/libltdl/ltdl.h
-endif
-endif
-
 sysconf_DATA = gnashrc gnashpluginrc
 
 edit = sed \
@@ -45,7 +37,7 @@
 
 # If we are using an installable libltdl, then it needs to be built
 # before libgnash, as libgnash depends on it for extensions.
-pkglib_LTLIBRARIES =  $(LIBLTDLLIB) libgnashbase.la
+pkglib_LTLIBRARIES = libgnashbase.la
 
 libgnashbase_la_CPPFLAGS = -DPLUGINSDIR=\"$(pluginsdir)\" \
 	-DSYSCONFDIR=\"$(sysconfdir)\" \
@@ -121,7 +113,6 @@
 	$(NULL)
 
 noinst_HEADERS = \
-	$(LIBLTDLHEAD) \
 	SimpleBuffer.h \
 	extension.h \
 	GnashException.h \
@@ -167,21 +158,6 @@
 
 libgnashbase_la_LDFLAGS = -release $(VERSION)
 
-if !LIBLTDL2
-libltdlc_la_CPPFLAGS = $(LTDLINCL) -I$(top_srcdir)/libltdl
-if CONVENIENCE_LTDL
-noinst_LTLIBRARIES = libltdlc.la
-libltdlc_la_SOURCES = $(top_srcdir)/libltdl/ltdl.c
-endif
-
-libltdl_la_SOURCES = $(top_srcdir)/libltdl/ltdl.c
-libltdl_la_LDFLAGS = -no-undefined -version-info 4:4:1
-libltdl_la_LIBADD = $(LIBADD_DL)
-
-if WIN32
-  libltdl_la_LDFLAGS += -static
-endif
-endif
 CLEANFILES = libltdl.la libltdlc.la gnashrc gnashpluginrc
 
 # Rebuild with GCC 4.x Mudflap support
--- gnash-0.8.3/libmedia/Makefile.am.orig	2008-05-15 17:38:39.000000000 +0200
+++ gnash-0.8.3/libmedia/Makefile.am	2008-06-20 07:50:58.311893357 +0200
@@ -67,7 +67,6 @@
 	$(CURL_LIBS) \
 	$(OPENGL_LIBS) \
 	$(LIBLTDL) \
-	$(LIBLTDLHEAD) \
 	$(BOOST_LIBS) \
 	$(PTHREAD_LIBS) \
 	$(NULL)
